lib/Room.php

```diff
@@ -580,9 +580,10 @@ public function removeUser(IUser $user) {
 	/**
 	 * @param Participant $participant
 	 */
-	public function removeParticipantBySession(Participant $participant) {
+	public function removeParticipantBySession(Participant $participant, $reason) {
```

Comment:
`$reason` should have the type defined, i.e. `string $reason`.

Reply:
I would prefer a bool instead of magic strings

Reply:
magic strings allow for more than two states if the reason needs to be expanded in the future to more than just leave and remove.